ansible/roles/ewf-app-config/files/httpd/conf.d/ewf_perl.conf

```diff
@@ -0,0 +1,51 @@
+#
+# Mod_perl incorporates a Perl interpreter into the Apache web server,
+# so that the Apache web server can directly execute Perl code.
+# Mod_perl links the Perl runtime library into the Apache web server
+# and provides an object-oriented Perl interface for Apache's C
+# language API.  The end result is a quicker CGI script turnaround
+# process, since no external Perl interpreter has to be started.
+#
+
+LoadModule perl_module modules/mod_perl.so
+#PerlModule Apache2
+#PerlModule Apache::compat
+
+PerlRequire /etc/httpd/conf/ewf_startup.pl
+PerlRequire /home/ewf/MODULES/Apache/CacheControlHeadersOutputFilter.pm
```

Comment:
Does this `/home/ewf/MODULES/Apache/CacheControlHeadersOutputFilter.pm` exist on the current server home directory output?

Reply:
Just checked and it does, it's also in the MODULES file in the app bundle